### DISCUSSION TITLE
Github

### DISCUSSION THREAD

**Discussion by BloodDValz** [community/community#125318] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

I am new and wanted to learn about Github. What will be the best way to learn Github.

**Comment by pedrolemos4** [comment]
I have not personally used it but you have got this repository some say it's good 'skills/introduction-to-github'.
You can also check Github's Youtube channel and/or other known youtubers.

**Comment by judahpaul16** [comment]
The best way to learn IMO is to start using it. This video is good: https://www.youtube.com/watch?v=8Dd7KRpKeaE&pp=ygUPZ2l0aHViIHR1dG9yaWFs
